api/src/main/java/org/openmrs/api/ProviderService.java

```diff
@@ -224,6 +224,15 @@ public List<Provider> getProviders(String query, Integer start, Integer length,
 	 */
 	public ProviderAttributeType getProviderAttributeType(Integer providerAttributeTypeId);
 	
+	/**
+	 * Get a provider attribute type by it's name
+	 *
+	 * @param name the name of the provider attribute type
+	 * @return the provider attribute type for the given uuid
+	 * <strong>Should</strong> get the provider attribute type by it's name
+	 */
+	public ProviderAttributeType getProviderAttributeTypeByName(String name);
```

Comment:
Shouldn't you have the @Authorized annotation on this?

Reply:
added!